web/src/components/dashboard/OpportunityCard.test.tsx

```diff
@@ -111,6 +119,17 @@ describe("OpportunityCard", () => {
     expect(mockAnalytics.event.for_you_card_unhide_button.click.unhide_card).not.toHaveBeenCalled();
   });
 
+  it("fires opportunity_card analytics when title with link is clicked", () => {
+    render(
+      <OpportunityCard
+        opportunity={generateOpportunity({ name: "fundingName", urlSlug: "slug" })}
+        urlPath="funding"
+      />
+    );
+    fireEvent.click(screen.getByText("fundingName"));
+    expect(mockAnalytics.event.opportunity_card.click.go_to_opportunity_screen).toHaveBeenCalledTimes(1);
```

Comment:
is there a reason for the explicit `toHaveBeenCalledTimes(1)` instead of just `toHaveBeenCalled`?

Reply:
This might be a little excessive, but given that we also want to validate the integrity of the data we're capturing, maybe it makes sense to validate this is only called once?

Reply:
ok

Reply:
@jphechter @cbrunefearless It's fair, I guess I feel like it doesn't matter much so I'd rather just keep with the pattern. Unless we should change the other analytics event tests to be CalledTimes(1) also?